.env.example

```
@@ -11,6 +11,13 @@ BLOCKSCOUT_BS_LIGHT_TIMEOUT=20.0
 BLOCKSCOUT_BS_TIMEOUT=120.0
 BLOCKSCOUT_BENS_TIMEOUT=30.0
 BLOCKSCOUT_CHAINSCOUT_TIMEOUT=15.0
+
+# PRO API configuration endpoint — determines the authoritative set of supported chains and their URLs.
+# No API key required. Override only for testing against a non-production PRO API instance.
+BLOCKSCOUT_PRO_API_CONFIG_URL="https://api.blockscout.com/api/json/config"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)